Resources/Prototypes/Catalog/Fills/lockers.yml

```diff
@@ -56,21 +56,105 @@
   id: LockerBoozeFilled
   suffix: Filled
   parent: LockerBooze
+  components:
+  - type: StorageFill
+    contents:
+      - name: ClothingOuterArmorBulletproof
+        prob: 0.5
+      - name: ShotgunSawn
+        prob: 0.8
+      - name: DrinkShaker
+        prob: 1
+      - name: ClothingEyesGlassesBeer
+        prob: 1
+      - name: DrinkBeer
+        prob: 0.5
+      - name: DrinkBottleBeer
+        prob: 0.5
 
 - type: entity
   id: LockerQuarterMasterFilled
   suffix: Filled
   parent: LockerQuarterMaster
+  components:
+  - type: StorageFill
+    contents:
+      - name: ClothingNeckCloakQm
+        prob: 1
+      - name: ClothingHeadsetCargo
+        prob: 1
+      - name: PlushieLizard
+        prob: 0.1
+      - name: ClothingHandsGlovesColorBrown
+        prob: 1
+      - name: ClothingOuterSuitFire
+        prob: 0.5
+      - name: ClothingShoesColorBrown
+        prob: 0.7 
+      - name: ClothingHeadHatCargosoft
+        prob: 0.8
+      - name: SupplyRequestComputerCircuitboard
+        prob: 1
+      - name: SupplyComputerCircuitboard
+        prob: 1
 
 - type: entity
   id: LockerCaptainFilled
   suffix: Filled
   parent: LockerCaptain
-
+  components:
+  - type: StorageFill
+    contents:
+      - name: NukeDisk
+        prob: 1
+      - name: CaptainPDA
```

Comment:
Why does the captain's PDA spawn inside his locker?

Reply:
It's captain's spare ID but since we don't want to cause bloat, and have no PDA painter, it is replaced with a PDA with a nameless captain's ID inside

Reply:
Though if that seems weird I can replace it by making a new "Captain's spare ID"